README.md

```diff
@@ -26,8 +26,7 @@ This repository contains packages and applications for Recall, written in JavaSc
 ```
 js-recall/
 ├── apps/                    # Application packages
-│   ├── portal/             # Main web application
```

Comment:
So the bulk of the changes in this PR are just the faucet, portal, and registration folders (and their contents) have been deleted.